cmd/replay/main.go

```diff
@@ -51,7 +51,13 @@ func getBlockHeaderByHash(hash common.Hash) *types.Header {
 	return header
 }
 
-type WavmChainContext struct{}
+type WavmChainContext struct {
+	chainConfig *params.ChainConfig
+}
+
+func (c WavmChainContext) Config() *params.ChainConfig {
```

Comment:
Let's remove from ProduceBlock (and Advanced) the chainConfig param and use the one from chainContext.

Reply:
Done.